qgepplugin/processing_provider/qgep_swmm_commands.py

```diff
@@ -0,0 +1,37 @@
+# This template file can be used to run the qgep-swmm functionalities 
+# outside from QGIS
+
+from QgepSwmm import QgepSwmm
+
+# Title of the simulation
+TITLE = "titre"
+# Name of the service used to connect to the database
+PGSERVICE = "pg_qgep"
+#State of the simulation [current, planned]
+STATE = "current"
+# Path of the input INP file
+INPUT = "C:/temp/input.inp"
+# Path of the input template INP file
+TEMPLATEINP = "C:/temp/gruner/out/swmm_template.inp"
```

Comment:
If I don't have a folder "C:/temp/gruner/out/" this will fail?
So do I need to adjust that to my system?

Reply:
Yes, these parameters must be adjusted to your system. But, this file will not be used by 99% of the users. Only advanced users and developers might want to run qgep-swmm without QGIS. This is why I'm not sure that we need to keep it in Git. Maybe it is better to delete it?